Pock/Preferences/Panes/GeneralPreferencePane/GeneralPreferencePane.swift

```diff
@@ -26,10 +26,10 @@ final class GeneralPreferencePane: NSViewController, Preferenceable {
     
     /// Preferenceable
     let toolbarItemTitle: String   = "General"
-    let toolbarItemIcon:  NSImage  = NSImage(named: NSImage.Name("pock-icon"))!
-    
+	let toolbarItemIcon:  NSImage  = NSImage(named: .preferencesGeneral)!
```

Comment:
You should prefer using the system icon for this. It's what users are used to. And it looks weird to have the app icon there.